### PR TITLE
[RMFT June 2022] Alex's Lab

### PR DESCRIPTION
**Pull request by Virgule09**

### CODE REVIEW

**Comment by Virgule09** [comment]
I took a break, ate diner and finally everything clicked. I was using the useState hook multiple times when I was supposed to reuse the function.
Tried it with the delete functionality and it worked. From then on it was just a formality.
During the whole day I had no difficulties understanding the concept so this was very frustrating, I am happy that I managed to get it done, finally ! :)
PS: I believe my table is not working, or maybe it is just CSS, but I must confess that I did not care too much about this. But I already know that I could have done better there, sorry about that.

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,62 @@
...
 function App() {
+
+  const [contactsArr, setContact] = useState(contacts.slice(0, 5));
+  const addRandomContact = () => {
+    const randomContact = contacts[Math.floor(Math.random() * contacts.length)];
+    setContact((prevContacts) => {
+      const newContacts = [...prevContacts, randomContact]
+      return newContacts
+    })
+  }
```
Comment (TA-Remote) [inline src/App.js:14]: Last step for this lab would be to display list without repeated contacts :) 

Clue: you might want to create new array that removes contacts that already being displayed, and get random contact from that new list

**Review by TA-Remote — COMMENTED** [review]
The most important part is that you didn't give up 😄  and it clicked eventually, well done!

File: src/App.js (hunk 1 above)
Comment (Virgule09) [reply src/App.js:14]: Right !!! I completely forgot about this :/

**Pull request closed without merging**